### PR TITLE
GH-449: Removal of BroadcastTrigger

### PR DESCRIPTION
**Pull request by bertllll**

### CODE REVIEW

File: masq_lib/src/test_utils/mock_websockets_server.rs [hunk 1]
```diff
@@ -123,9 +124,16 @@ impl MockWebSocketsServer {
             }
             log(do_log, index, "Entering background loop");
             loop {
+                log(do_log, index, "Checking for fire-and-forget messages");
+                self.handle_potential_fire_and_forget_messages(
```
Comment (bertllll) [inline masq_lib/src/test_utils/mock_websockets_server.rs]: renaming? loose_fire_and_forget_msgs_if_they_are_at _the_tip_of_the_queue

File: masq_lib/src/test_utils/mock_websockets_server.rs [hunk 2]
```diff
@@ -429,253 +383,170 @@ fn log(log: bool, index: u64, msg: &str) {
...
-        // connection.receive() -> error: No more Broadcasts available
+        // connection.receive() -> Broadcast 3
 
-        //Content of those messages is practically irrelevant because it's not under the scope of this test.
-        //Also, a lot of lines could be highlighted with text like this "TESTED BY COMPLETING THE TASK - NO ADDITIONAL ASSERTION NEEDED",
-        //but it may have made the test (even) harder to read.
+        //Content of those messages is practically irrelevant because it's not in the scope of this test.
 
-        //Lists of messages used in this test
+        //you should view some lines in this test as if highlighted with text like "TESTED BY COMPLETING THE TASK - NO ADDITIONAL ASSERTION NEEDED"
```
Comment (bertllll) [inline masq_lib/src/test_utils/mock_websockets_server.rs]: Consider certain lines of this test as if they were highlighted with text saying...

File: masq_lib/src/test_utils/mock_websockets_server.rs [hunk 3]
```diff
@@ -429,253 +383,170 @@ fn log(log: bool, index: u64, msg: &str) {
...
-            .queue_response(conversation_number_two_response)
+            .queue_response(conversation_number_one_response.clone().tmb(1))
+            .queue_response(conversation_number_two_response.clone().tmb(2))
             .queue_response(broadcast_number_one)
             .queue_response(broadcast_number_two)
-            .queue_response(broadcast_number_three)
-            .queue_response(broadcast_number_four)
-            .queue_response(broadcast_number_five)
             .queue_response(conversation_number_three_response)
-            .queue_response(broadcast_number_six);
```
Comment (bertllll) [inline masq_lib/src/test_utils/mock_websockets_server.rs:572]: Add an assertion even for the third conversational response. Look at those two else.

File: node/src/database/db_initializer.rs [hunk 4]
```diff
@@ -368,6 +368,37 @@ impl DbInitializerReal {
...
+    }
+
+    fn validate_schema_version(obtained_s_v: &str) -> usize {
+        obtained_s_v.parse::<usize>().unwrap_or_else(|_| {
+            panic!(
+                "Database version should be purely numeric, but was: {}",
+                obtained_s_v
+            )
+        })
+    }
```
Comment (bertllll) [inline node/src/database/db_initializer.rs]: Examine why this code returns under the lights, did I make a mistake in the master merge?

File: masq_lib/src/test_utils/mock_websockets_server.rs [hunk 5]
```diff
@@ -429,253 +383,171 @@ fn log(log: bool, index: u64, msg: &str) {
...
         // Queue:
         // Conversation 1
         // Conversation 2
         // Broadcast 1
         // Broadcast 2
-        // Broadcast 3
-        // Broadcast 4
         // Conversation 3
-        // Conversation 4
         // Broadcast 5
```
Comment (dnwiebe) [inline masq_lib/src/test_utils/mock_websockets_server.rs]: Change to Broadcast 3

**Review by dnwiebe — APPROVED** [review]

**Pull request merged**